src/stdlib.sol

```diff
@@ -61,6 +61,28 @@ abstract contract stdCheats {
         vm_std_cheats.deal(who, give);
         vm_std_cheats.startPrank(who, origin);
     }
+
+    // Deploys a contract by fetching the contract bytecode from
+    // the artifacts directory
+    function transcribe(string memory what, bytes memory args)
```

Comment:
`transcribe` feels too abstract IMO, what about `deployCode` which nicely parallels the `getCode` method used?

Reply:
ye if we can change it to `deployCode` im 1000% on board this is great. I like maker naming when it isnt easily captured what it does without being overly verbose but `deployCode` isnt too verbose